python/param_io.cpp

```diff
@@ -0,0 +1,114 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <fstream>
+#include <iomanip>
+
+#include <arbor/cable_cell.hpp>
+#include <arbor/morph/primitives.hpp>
+
+#include <arborio/jsonio.hpp>
+
+#include "error.hpp"
+#include "strprintf.hpp"
+
+namespace pyarb {
+void register_param_loader(pybind11::module& m) {
+    m.def("load_default_parameters",
+          [](std::string fname) {
```

Comment:
Instead of implementing these as lambdas, they could be implemented above as stand alone functions (with return types clearly defined).

Reply:
I'll make the change.